timescaledb/how-to-guides/ingest-data/about-ingesting.md

```diff
@@ -0,0 +1,11 @@
+# Ingesting data from other sources
```

Comment:
**question:** Is this page strictly necessary? I know it's in our how-to guide template, but there's so little conceptual information here that it's basically a repeat of the index page. Might that come across as a little annoying to anyone who navigates here to learn more?

Reply:
My thought was that we really should come up with something to write here, actually 😂